packages/codegen-ui/lib/types/studio-types.ts

```diff
@@ -570,6 +543,21 @@ export type DataStoreDeleteItemAction = {
   };
 };
 
+export type MutationAction = {
+  action: 'Amplify.Mutation';
+  parameters: {
+    state: MutationActionSetStateParameter;
+  };
+};
+
+export type MutationActionSetStateParameter = {
+  componentName: string;
+  property: string;
+  set: StudioComponentProperty;
+} & CommonPropertyValues;
+
```

Comment:
Can probably remove the CommonPropertyVAlues here, since we made that change in the internal representation as well.

Reply:
Addressed in the latest commit.